### ISSUE TITLE
Too many WebMediaPlayers already in existence

### ISSUE THREAD

**Issue by jkuester** [medic/cht-conf-test-harness#185]
_(Created from draft issue on [Enketo Uplift AT Issues board](https://github.com/orgs/medic/projects/120/views/1) for better linkage and paper-trail.  THIS ISSUE ONLY AFFECTS THE 4.x BRANCHES, NOT ANY RELEASED VERSION)_

## Kenn

I'm seeing this error in the console when running many tests in succession:
`Blocked attempt to create a WebMediaPlayer as there are too many WebMediaPlayers already in existence. See crbug.com/1144736#c27`

I can also reproduce this by launcihng the muso patient_assessment form like 100 times. 

I unload the form at the end of each test.
Can be reproduced using npx `mocha test/tasks/treatment_followup_under_5.spec.js --timeout 30000`

## 2022-05-23 (Josh)

TLDR here is that I think the harness needs to be resetting the page between forms. Normally when running in the CHT webapp, the Angular framework will clean up the page DOM after the form is submitted and a new component is loaded (unfortunately, the `EnektoFormModel.unload` does not actually clear the DOM). So, with the test harness, the page is never getting reset when calling `harness.clear` and the page contents continue to grow as more tests run.

I have done extensive testing of this issue to verify that it is only reproducible in the test harness and not when running cht-core in production.  I was able to reproduce [similar behavior in cht-core](https://github.com/medic/cht-core/issues/7627) but it was isolated to only forms with both the `countdown-widget` and the `geopicker` widget.  That issue seems related, but more narrow since with the test harness I can reproduce the error with forms that only have the `countdown-widget` (and do not contain the `geopicker`).  However, #7627 does not seem to be the root cause of the issue in the test harness since that issue also affects previous versions of the CHT (all the way back to at least `3.10.5`) while the `treatment_followup_under_5.spec.js` test worked fine in the old version of the test harness + CHT.

I have opened [a PR](https://github.com/medic/cht-conf-test-harness/pull/173) for the test harness to reload the page when clearing the form.  This seems like kind of a brute force solution, but is at least a simple fix that works. 🤷 

Investigation Details:
- The `countdown-timer` is the origin of the WebMediaPlayer error because of the way it plays a sound when alerting the user.
  - One effect of the error is that new countdown timers will not play the sound alert when the timer completes. (Everything else seems to still behave normally).
- There are several very length threads out on crbug.com regarding this issue and apparently the Chromium developers have made several changes recently regarding the actual limit of these WebMediaPlayers 
  - Interestingly the Chromium version being pulled in for me by `puppeteer` (`93.0.x`) seems to be the version with the lowest limit for WebMediaPlayers at `75`.
  - My current Chromium version of `101.0.x` has a limit of `1000` WebMediaPlayers.
- Those above limits to affect the total number of countdown-widgets that a form can have. (So with Chromium 93, a form can only have a maximum of 75 countdown-widgets.)
  - This is not too concerning since that is a lot of countdown timers. The real problem in the production code is with [#7627](https://github.com/medic/cht-core/issues/7627) you have a cumulative effect where the timers from previous forms do not get cleaned up and eventually you will hit the limit.
